lab-james/lib/use-call.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.Game = function(title, genre) {
+  this.title = title;
+  this.genre = genre;
+};
+
+exports.PcGame = function(title, genre) {
+  Game.call(this, title, genre);
+  this.isForPc = true;
+};
+
+let mario64 = new exports.Game('Mario 64', 'Platformer');
```

Comment:
Generally you will want to create new objects like this in a separate index.js file, especially per lab instructions.